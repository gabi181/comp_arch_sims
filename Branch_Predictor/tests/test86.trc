32 5 4 1 global_history local_tables not_using_share
0x217DFBDC N 0x0035E730
0x67D27F38 N 0x57FF8C68
0x2B883C44 T 0x128E5AF8
0x217DFBDC T 0x0035E730
0x217DFBDC T 0x0035E730
0x2C9BA154 N 0x492C9B98
0x217DFBDC N 0x0035E730
0x2C9BA154 N 0x492C9B98
0x67D27F38 T 0x57FF8C68
0x36047F54 N 0x5B2BF7A8
0x2B883C44 T 0x128E5AF8
0x67D27F38 T 0x57FF8C68
0x67D27F38 N 0x57FF8C68
0x36047F54 N 0x5B2BF7A8
0x36047F54 T 0x5B2BF7A8
0x217DFBDC N 0x0035E730
0x67D27F38 N 0x57FF8C68
0x67D27F38 T 0x57FF8C68
0x2B883C44 N 0x128E5AF8
0x43F7AE64 T 0x26F6AE18
