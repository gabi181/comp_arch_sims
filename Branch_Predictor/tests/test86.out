0x217dfbdc N 0x217dfbe0
0x67d27f38 N 0x67d27f3c
0x2b883c44 N 0x2b883c48
0x217dfbdc N 0x217dfbe0
0x217dfbdc N 0x217dfbe0
0x2c9ba154 N 0x2c9ba158
0x217dfbdc N 0x217dfbe0
0x2c9ba154 N 0x2c9ba158
0x67d27f38 N 0x67d27f3c
0x36047f54 N 0x36047f58
0x2b883c44 N 0x2b883c48
0x67d27f38 N 0x67d27f3c
0x67d27f38 N 0x67d27f3c
0x36047f54 N 0x36047f58
0x36047f54 N 0x36047f58
0x217dfbdc N 0x217dfbe0
0x67d27f38 N 0x67d27f3c
0x67d27f38 N 0x67d27f3c
0x2b883c44 N 0x2b883c48
0x43f7ae64 N 0x43f7ae68
flush_num: 9, br_num: 20, size: 3173b
